tests/playwright/resources/squid-proxy/generate-certs.sh

```
@@ -0,0 +1,90 @@
+# If certificates for other hostnames/IPs are needed, modify the [alt_names]
+# section
+#
+# The certificate used by squid can be overriden when starting the container:
+# podman run -v ./pki/:/etc/pki/squid:z ...
+
+set -euo pipefail
+
+destDir=${1:-pki}
+baseName=${2:-squid}
+
+mkdir -p $destDir
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)